src/Translumo/MVVM/ViewModels/LanguagesSettingsViewModel.cs

```diff
@@ -34,6 +37,14 @@ public sealed class LanguagesSettingsViewModel : BindableBase, IAdditionalPanelC
 
         public TtsConfiguration TtsSettings { get; set; }
 
+        // NOTE: wfp doesnt update combobox source for non-static propertions or properties from non-singletone class 💀, I cant find another workaround
+        public static ObservableCollection<string> AvailableVoices { get; } = new();
```

Comment:
There is some problem with VM instances... I will investigate it further and come back with update